lib/audit/checksum.rb

```diff
@@ -3,12 +3,23 @@
 # Checksum validator code
 class Checksum
 
-  # FIXME: remove rubocop exception once we start filling these methods in
-  # rubocop:disable Lint/UnusedMethodArgument
-  # rubocop:disable Style/EmptyMethod:
-  # TODO: implement this;  we begin with a placeholder
-
-  def self.validate_disk(endpoint_name)
+  def self.validate_disk(endpoint_name, limit=Settings.c2m_sql_limit)
+    start_msg = "#{Time.now.utc.iso8601} CV validate_disk starting for #{endpoint_name}"
+    puts start_msg
+    Rails.logger.info start_msg
+    all_processable_copies = PreservedCopy.by_endpoint_name(endpoint_name).fixity_check_expired
```

Comment:
total quibble:  "pres_copies_to_validate" instead of "all_processable_copies" ?
- pres_copies_to_validate
- pres_copies_to_process
- pcs_to_validate
- pcs_w_expired_fixity_check
...

Reply:
well, the relation as defined would actually return all processable copies, but the `.count` and `.limit` methods chained onto `all_processable_copies` keep it from actually doing that when the final query is constructed.

Reply:
my quibble is more with "processable" because technically, we could process any PreservedCopy object;  these are the ones we've SELECTED to be processed based on fixity expiration.

Reply:
ah, yeah, i see, i figured it was the `all` you didn't like.

Reply:
in that case, i like you're last one, `pcs_w_expired_fixity_check`